index.html

```diff
@@ -139,55 +139,76 @@ <h3 class="project-title"> Multi-Post Stories <br> Gain+Glory</h3>
+                            <li class="skill-tag">HTML</li>
+                            <li class="skill-tag">CSS</li>
+                        </ul>
+                    </div>
                 </div>
-            </div>
-            <div class="skills">
-                <img src="assets/Frameworks.svg" alt="Frameworks">
-                <div>
-                    <h2 class="skill-title">Frameworks</h2>
-                    <ul>
-                        <li class="skill-tag">React.js</li>
-                        <li class="skill-tag">Ruby on Rails</li>
-                        <li class="skill-tag">RSpec</li>
-                        <li class="skill-tag">Capybara</li>
-                        <li class="skill-tag">Selenium</li>
-                    </ul>
+                <div class="skills">
+                    <img src="assets/Frameworks.svg" alt="Frameworks">
+                    <div>
+                        <h2 class="skill-title color1">Frameworks</h2>
+                        <ul>
+                            <li class="skill-tag">React.js</li>
+                            <li class="skill-tag">Ruby on Rails</li>
+                            <li class="skill-tag">RSpec</li>
+                            <li class="skill-tag">Capybara</li>
+                            <li class="skill-tag">Selenium</li>
+                        </ul>
+                    </div>
                 </div>
-            </div>
-            <div class="skills">
-                <img src="assets/Skills.svg" alt="Languages">
-                <div>
-                    <h2 class="skill-title">Skills</h2>
-                    <ul>
-                        <li class="skill-tag">Database Management</li>
-                        <li class="skill-tag">Version Control</li>
-                        <li class="skill-tag">CLI</li>
-                        <li class="skill-tag">Web Development</li>
-                        <li class="skill-tag">API Design</li>
-                    </ul>
+                <div class="skills">
+                    <img src="assets/Skills.svg" alt="Languages">
+                    <div>
+                        <h2 class="skill-title color1">Skills</h2>
+                        <ul>
+                            <li class="skill-tag">Database Management</li>
+                            <li class="skill-tag">Version Control</li>
+                            <li class="skill-tag">CLI</li>
+                            <li class="skill-tag">Web Development</li>
+                            <li class="skill-tag">API Design</li>
+                        </ul>
+                    </div>
                 </div>
             </div>
-        </div>
+        </section>
+        <section class="contact">
+                <h2 class="primary color1">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+            <form action="" class="form" action="https://formspree.io/f/mwkjgvpr"
```

Comment:
- [ ] Great job adding formspree. But when I try submitting the form, I get an error. Kindly make sure your link is correct.
![image](https://github.com/BHObserver/portfolio/assets/94551484/c15060cb-f4aa-4ffa-b1d0-5188ce9a6126)